scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -1188,11 +1188,22 @@ <h6 class="card-subtitle mb-2 text-muted"><span class="badge badge-warning">Part
               </td>
             </tr>
           {% endfor %}
-        {% endif %}
-      </tbody>
+        </tbody>
+      {% elif variant.genes %} <!-- more than 5 genes in this SV -->
+        <thead>
+          <th>{{ variant.genes|length }} affected gene(s)</th>
+        </thead>
+        <tbody>
+          <td>
+            {% for gene in variant.genes|sort(attribute='common.hgnc_symbol') %} <!-- Display all genes sorted by symbol -->
+              <a href="{{ url_for('genes.gene', hgnc_id=gene.hgnc_id) }}">
+              {{ gene.common.hgnc_symbol if gene.common else gene.hgnc_id }}
+              </a>&nbsp;
```

Comment:
Oh! Cool, where does it decide to break for a long line? I thought it would kind of just go on out of the bounding box! How is this different from layouting with regular spaces? Taking the opportunity to learn here! 😁

Reply:
It won't be a long line because there are regular spaces (&nbsp;) between gene names. Looks like the screenshot in the PR description

Reply:
Let's keep it like this and show 3 genes instead